docs/ref/contrib/admin/index.txt

```diff
@@ -2901,6 +2901,29 @@ Templates can override or extend base admin templates as described in
     Path to a custom template that will be used by the admin site password
     change done view.
 
+.. attribute:: AdminSite.final_catch_all_view
+
+    .. versionadded:: 3.2
+
+    A boolean value that determines whether to add a final catch-all view to
+    the admin that redirects unauthenticated users to the login page. By
+    default, it is set to ``True``.
+
+    .. warning::
+
+        Setting this to ``False`` is not recommended as the view protects
+        against a potential model enumeration privacy issue.
+
+.. attribute:: AdminSite.append_slash
+
+    .. versionadded:: 3.2
+
+    A boolean value that determines whether the catch-all view provided by
+    :attr:`~.AdminSite.final_catch_all_view` should apply
+    :setting:`APPEND_SLASH` style URL normalization for authenticated staff
+    users. Only applies when :setting:`APPEND_SLASH` is ``True``. By default,
+    it is set to ``True``.
```

Comment:
I know that not all attributes are alphabetized, but can we move `AdminSite.append_slash` below `AdminSite.app_index_template`, and `AdminSite.final_catch_all_view` below `AdminSite.enable_nav_sidebar`.